tests/test_commands.py

```diff
@@ -1639,6 +1639,19 @@ def test_cluster_slaves(self, mock_cluster_resp_slaves):
         assert isinstance(mock_cluster_resp_slaves.cluster(
             'slaves', 'nodeid'), dict)
 
+    @skip_if_server_version_lt('3.0.0')
+    def test_readwrite(self, r):
+        assert bool_ok(r.readwrite())
```

Comment:
Why can we not just use `assert r.readwrite()`? Shouldn't the `bool_ok` response callback have already been called?

Reply:
@andymccurdy Sorry for the delay. PR updated